libcxx/include/__chrono/year_month.h

```diff
@@ -92,6 +92,26 @@ _LIBCPP_HIDE_FROM_ABI constexpr
 year_month operator-(const year_month& __lhs, const years& __rhs) noexcept
```

Comment:
Note to self: mark those operators as `inline` explicitly. Relying on `constexpr` for `inline` semantics is really not a good idea.

Reply:
https://github.com/llvm/llvm-project/issues/75227